Project/Source/Components/UI/ComponentTransform2D.h

```diff
@@ -13,8 +13,45 @@
 #define TESSERACT_ENGINE_API __declspec(dllexport)
 #endif
 
-//TODO THIS CLASS CURRENTLY ONLY WORKS FOR OBJECTS THAT USE THE CENTER OF THE SCREEN AS THE CENTER OF COORDINATES,
-//NEITHER PIVOT NOR ANCHOR ARE TAKEN INTO ACCOUNT IN ANY WAY, NOR ARE THEY MODIFIABLE LIKE THEY WOULD IN UNITY
+// ANCHOR PRESETS CREATED BUT NOT WORKING -> WOLUD BE FIXED IN OTHER BRANCH
+// ANCHOR ARE TAKEN INTO ACCOUNT IN ANY WAY, NOR ARE THEY MODIFIABLE LIKE THEY WOULD IN UNITY
+
+struct AnchorPreset {
+	enum class AnchorPresetType {
+		// Presets
+		TOP_VERTICAL_LEFT_HORIZONTAL,
+		TOP_VERTICAL_CENTER_HORIZONTAL,
+		TOP_VERTICAL_RIGHT_HORIZONTAL,
+		TOP_VERTICAL_STRETCH_HORIZONTAL,
+
+		MIDDLE_VERTICAL_LEFT_HORIZONTAL,
+		MIDDLE_VERTICAL_CENTER_HORIZONTAL,
+		MIDDLE_VERTICAL_RIGHT_HORIZONTAL,
+		MIDDLE_VERTICAL_STRETCH_HORIZONTAL,
+
+		BOTTOM_VERTICAL_LEFT_HORIZONTAL,
+		BOTTOM_VERTICAL_CENTER_HORIZONTAL,
+		BOTTOM_VERTICAL_RIGHT_HORIZONTAL,
+		BOTTOM_VERTICAL_STRETCH_HORIZONTAL,
+
+		STRETCH_VERTICAL_LEFT_HORIZONTAL,
+		STRETCH_VERTICAL_CENTER_HORIZONTAL,
+		STRETCH_VERTICAL_RIGHT_HORIZONTAL,
+		STRETCH_VERTICAL_STRETCH_HORIZONTAL,
```

Comment:
Should we have a dropdown in the inspector to select one of those, instead of editing the floats manually? or both options?

Reply:
For this, I put a combo for the presets in the inspector where you can select if you want a custom anchoring or one of the presets, but it was implemented in the branch `feature/CU-eazafr_UI_AnchoringItems` and right now I have to fixed and put it in a better way.

I put a gif about it to see how looks at the moment.
![Debug 2021 05 06 - 02 16 05 02](https://user-images.githubusercontent.com/25251729/117225278-3e221f00-ae12-11eb-84b7-1cb560c320a8.gif)